src/functional/geometry/convex_hull.zig

```diff
@@ -103,6 +214,15 @@ pub const ConvexHull = struct {
             try hull.append(point);
         }
     }
+
+    /// Returns the length of the `ConvexHull`. Since the first and final element are repeated,
```

Comment:
Maybe make the part of the doc comment about subtracting two a comment for the last line of code as it seems to be an implementation detail and thus not information the user needs to consider.

Reply:
fixed